packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -1,10 +1,17 @@
 /**
- * CSS Custom Properties
- *
- * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-popover-z-index: Sets the z-index value for the component.
- */
+* CSS Custom Properties
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-popover-background-color: Specifies the background color of the component.
+* @prop --calcite-popover-border-color: The border color of the component.
+* @prop --calcite-popover-border-inline-end-color: The border color of the component.
```

Comment:
I think we can use a single border color and a single corner radius, not split by side.